pkg/controllers/provisioning/scheduling/topologygroup.go

```diff
@@ -94,7 +94,7 @@ func (t *TopologyGroup) Get(pod *v1.Pod, podDomains, nodeDomains sets.Set) sets.
 	case TopologyTypePodAntiAffinity:
 		return t.nextDomainAntiAffinity(podDomains)
 	default:
-		return sets.NewSet()
+		panic(fmt.Sprintf("Unrecognized topology group type: %s", t.Type))
```

Comment:
These types are wholly controlled by programmers, so this can only panic if the program has entered an undefined state. Rather than continuing to execute with undefined behavior, it's safer to panic.